src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -344,14 +344,14 @@ const NavBar: React.FC<NavBarPropsType> = ({ redirectionUrl }) => {
         </div>
       </header>
 
-      <SchedulePageModal isVisible={isEventScheduleVisible} />
-
       <div
         className={`schedule-dropdown-backdrop ${
           isEventScheduleVisible ? "show" : ""
         }`}
         onClick={hideEventSchedule}
-      />
+      >
+        <SchedulePageModal isVisible={isEventScheduleVisible} />
+      </div>
```

Comment:
[FYI] I nested these so that they aren't in the same 'stacking context', which simplifies our `z-index` layering.

[Future Cleanup] We can probably remove the explicit `z-index` layer on the `SchedulePageModal` component now I think, since I believe it will inherit from `.schedule-dropdown-backdrop`'s layer now.